src/main/resources/logback-spring.xml

```diff
@@ -18,7 +18,7 @@
         </root>
     </springProfile>
 
-    <springProfile name="default">
+    <springProfile name="!local">
```

Comment:
if `default` works I prefer this... which is more generic

Reply:
same as other projects already in use. If not mandatory I'd suggest not to apply any further update